lib/ansible/modules/network/ios/ios_l2_interface.py

```diff
@@ -155,7 +155,9 @@ def interface_is_portchannel(name, module):
 def get_switchport(name, module):
     config = run_commands(module, ['show interface {0} switchport'.format(name)])[0]
     mode = re.search(r'Administrative Mode: (?:.* )?(\w+)$', config, re.M).group(1)
-    access = re.search(r'Access Mode VLAN: (\d+)', config).group(1)
+    access = re.search(r'Access Mode VLAN: (\d+)', config)
```

Comment:
re.search will return None if no match is found. So other pararmeters which are using `group()` on the same line might return NoneType error as well for other cases.
Can you please do this change for the other params also?

Reply:
I have done the necessary modifications as it's better to handle all match objects once for all.